lib/pxenforcer.js

```diff
@@ -73,6 +83,50 @@ class PxEnforcer {
         }
     }
 
+    isTelemetryCommand(req) {
+
+        const headerVal = req.headers[this._config.TELEMETRY_COMMAND_HEADER];
+
+        if(!headerVal) {
+            return false;
+        }
+
+        this.logger.debug('Received command to send enforcer telemetry');
+
+        //base 64 decode
+        const decodedString = Buffer.from(headerVal, 'base64').toString();
+
+        //value is in the form of timestamp:hmac_val
```

Comment:
this is a better comment!

Reply:
But I would instead deconstruct the variables like so:
`const [timestamp, hmac] = splittedValue;` and then it's more readable through the rest of the code.

Reply:
changed